storybook-static/index.json

```diff
@@ -0,0 +1 @@
+{"v":4,"entries":{"configure-your-project--docs":{"id":"configure-your-project--docs","title":"Configure your project","name":"Docs","importPath":"./src/stories/Configure.mdx","storiesImports":[],"type":"docs","tags":["unattached-mdx","docs"]},"counter--default":{"type":"story","id":"counter--default","name":"Default","title":"Counter","importPath":"./src/app/stories/Counter.stories.js","tags":["story"]},"example-button--docs":{"id":"example-button--docs","title":"Example/Button","name":"Docs","importPath":"./src/stories/Button.stories.js","type":"docs","tags":["autodocs","docs"],"storiesImports":[]},"example-button--primary":{"type":"story","id":"example-button--primary","name":"Primary","title":"Example/Button","importPath":"./src/stories/Button.stories.js","tags":["autodocs","story"]},"example-button--secondary":{"type":"story","id":"example-button--secondary","name":"Secondary","title":"Example/Button","importPath":"./src/stories/Button.stories.js","tags":["autodocs","story"]},"example-button--large":{"type":"story","id":"example-button--large","name":"Large","title":"Example/Button","importPath":"./src/stories/Button.stories.js","tags":["autodocs","story"]},"example-button--small":{"type":"story","id":"example-button--small","name":"Small","title":"Example/Button","importPath":"./src/stories/Button.stories.js","tags":["autodocs","story"]},"example-header--docs":{"id":"example-header--docs","title":"Example/Header","name":"Docs","importPath":"./src/stories/Header.stories.js","type":"docs","tags":["autodocs","docs"],"storiesImports":[]},"example-header--logged-in":{"type":"story","id":"example-header--logged-in","name":"Logged In","title":"Example/Header","importPath":"./src/stories/Header.stories.js","tags":["autodocs","story"]},"example-header--logged-out":{"type":"story","id":"example-header--logged-out","name":"Logged Out","title":"Example/Header","importPath":"./src/stories/Header.stories.js","tags":["autodocs","story"]},"example-page--logged-out":{"type":"story","id":"example-page--logged-out","name":"Logged Out","title":"Example/Page","importPath":"./src/stories/Page.stories.js","tags":["story"]},"example-page--logged-in":{"type":"story","id":"example-page--logged-in","name":"Logged In","title":"Example/Page","importPath":"./src/stories/Page.stories.js","tags":["play-fn","story"]}}}
```

Comment:
storybook-static should be added to gitignore, please, update you gitignore file 
https://dev.coko.foundation/?path=/docs/recipes-gitignore--page